js/qmwp-bargraph-gauge-scatterplot-timeline_charts.js

```diff
@@ -236,7 +236,7 @@ AnalyzeChart = function () {
                 var inputVar = getValueForDate(cause.timeSeries, date);
                 var outputVar = getValueForDate(effect.timeSeries, date);
 
-                if (inputVar && outputVar) {
+                if (typeof inputVar == 'number' && typeof outputVar == 'number') {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1109033051/issues/source?bid=2836188&fileBranchId=2895618#l239)